srcts/src/components/card.ts

```diff
@@ -167,6 +156,21 @@ class Card {
     }
   }
 
+  private _trapFocusExit(event: KeyboardEvent): void {
+    if (!(event instanceof KeyboardEvent)) return;
+    if (event.key !== "Tab") return;
+    if (event.shiftKey) return;
+
+    // We have to check every time because the card contents may have changed
+    const focusableElements = getAllFocusableChildren(this.container);
+    const lastFocusable = focusableElements[focusableElements.length - 1];
+
+    if (event.target !== lastFocusable) return;
+    // If tabbing forwards out of the card, return to close button
+    event.preventDefault();
+    this.overlay.anchor.focus();
+  }
```

Comment:
Let's make sure this is somewhat responsive with a `card(plotly::plot_ly(x = rnorm(1000), y = rnorm(1000))`

Reply:
I only notice a lag at around `rnorm(1e5)` which doesn't really have anything to do with the card javascript.